### ISSUE TITLE
Why is the Leaderboard result about zero-shot classification lower than paper

### ISSUE THREAD

**Issue by TangYuan96** [qizekun/ReCon#11]
First, thanks for sharing so good work!

Why is the Leaderboard result about zero-shot classification lower than paper?

   zero-shot classification on  MN10：  81.6（ in paper）> 75.6 (inLeaderboard )
  
   zero-shot classification on MN40： 66.8（ in paper）> 61.7 (inLeaderboard )

Look forward to your reply~

**Comment by qizekun** [comment]
I'm very sorry for the inconvenience caused. MN40: 66.8% and MN10: 81.6% were zero-shot tested on the complete ModelNet dataset (i.e., train+test), but the more appropriate comparison should actually be done only on the test set. We have made the necessary corrections in the latest arXiv paper version and on the Leaderboard.

Also, we noticed your previous issue and have uploaded ModelNet10 to [Google Drive](https://drive.google.com/drive/folders/1DGoHY0G29UKlLcExzcuvrJcg5GlbCYdO?usp=drive_link) for you to use.

We sincerely appreciate your interest in our work!
